tmt/export.py

```diff
@@ -356,6 +352,27 @@ def export_to_nitrate(test):
         nitrate_case.identifier), fg='magenta'))
 
 
+def return_markdown_file():
+    """ Return path to the markdown file """
+    files = '\n'.join(os.listdir())
+    reg_exp = r'.+\.md$'
+    num_md_files = len(re.findall(reg_exp, files, re.M))
+    fail_message = "in the current working directory.\n" \
+                   "Manual steps couldn't be exported"
+    if num_md_files == 1:
+        md_path = os.getcwd() + '/' + \
+            re.search(reg_exp, files).group(0)
+    elif num_md_files == 0:
```

Comment:
```suggestion
    md_files = re.findall(reg_exp, files, re.M)
    fail_message = "in the current working directory.\n" \
                   "Manual steps couldn't be exported"
    if len(md_files) == 1:
        md_path = os.path.join(os.getcwd(), md_files[0])
    elif len(md_files) == 0:
```
The problem with failing test comes from the fact that you are not using `re.M` for the second search which makes handling of `$` different. The order of files from `os.listdir()` could be different between systems so that's probably why it fails only on EPEL. I think rather than matching again, you should reuse the results of `re.findall`. Also I'd suggest using `os.path.join()` rather than string concatenation.